package.json

```diff
@@ -98,8 +99,7 @@
     "ramda": "0.29.0",
     "rudder-component-cookie": "0.0.1",
     "serialize-javascript": "6.0.1",
-    "storejs": "2.0.5",
-    "uuid": "8.3.2"
```

Comment:
this was needed for v1.1 I believe

Reply:
In v1.1, we use `@lukeed/uuid` instead of `uuid` now.

Reply:
I see we have patch ed the dependency that was using it ok!